packages/core/src/App/app.jsx

```diff
@@ -99,6 +100,7 @@ const AppWithoutTranslation = ({ root_store }) => {
                                     <BinaryBotIFrame />
                                     <AppToastMessages />
                                 </PlatformContainer>
+                                <MockDialog />
```

Comment:
```suggestion
                                {is_mock_server_enabled && <MockDialog />}
```
@yashim-deriv How about adding a feature flag for it? 🤔

Reply:
I am putting it behind a production check. It is not a feature so i'm probably not using the flag @farzin-deriv